src/Namotion.Reflection/Context/CachedType.cs

```diff
@@ -179,7 +179,7 @@ public CachedType[] OriginalGenericArguments
         public T? GetInheritedAttribute<T>()
             where T : Attribute
         {
-            return InheritedAttributes.OfType<T>().SingleOrDefault();
```

Comment:
Probably it's better to have SingleOrDefault<T> here to avoid regressions.

Reply:
Changed to `SingleOrDefault` semantics